.github/workflows/build.yml

```diff
@@ -87,8 +90,8 @@ jobs:
       flavor: ${{ matrix.flavor }}
       commit_id: ${{ needs.requirements.outputs.commit_id }}
       version: ${{ needs.requirements.outputs.version }}
-      # GitHub workflow workaround for bool
-      use_kms: ${{ needs.requirements.outputs.use_kms == 'true' }}
+      signing_env: ${{ inputs.signing_env }}
```

Comment:
This logic should move to `build_requirements.yml` as suggested in #3012. Therefore no additional information about KMS is required here I think. If the builder requires `USE_KMS` as environment we should leave the input as it is and base it on the value of `target` (or `environment` as stated above).